pkgs/build-support/build-sandbox/src/Makefile

```diff
@@ -1,19 +1,24 @@
 BINARIES = $(wildcard $(BINDIR)/*)
 WRAPPERS = $(subst $(BINDIR),$(out)/bin,$(BINARIES))
 
+OBJECTS = path-cache.o params.o setup.o
+CFLAGS = -g -Wall -std=gnu11 -DFS_ROOT_DIR=\"$(out)\"
+CXXFLAGS = -g -Wall -std=c++14 `pkg-config --cflags nix-main`
+LDFLAGS = -Wl,--copy-dt-needed-entries `pkg-config --libs nix-main`
```

Comment:
If I remove `--libs nix-main`, the linker can’t find the symbols for `std` anymore, what to put instead?

Reply:
Yeah, you probably also need to remove the linker flag and the cflags accordingly.

Reply:
If I remove both, I get missing symbols from `std`.

I don’t know what magic `CC` does to compile `sandbox.c`, but something misses when `--libs nix-main` is removed.